cloud-aws/src/main/resources/definitions/freeipa-minimal-policy.json

```diff
@@ -0,0 +1,106 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
```

Comment:
Do not we also need `ec2:DeleteTags` (with the same `ec2:ResourceTag/Cloudera-Resource-Name` restriction)?